### DISCUSSION TITLE
new ComfyUI version

### DISCUSSION THREAD

**Discussion by derpmagician** [comfyanonymous/ComfyUI#4437] [Q&A]
So i decided to run ComfyUI in a new computer with linux but got me the surprise that the file strucutre changed, there is a model_filemanager folder but

where should i put the models now?

![image](https://github.com/user-attachments/assets/bd5ee9ae-5bdd-41d1-8cca-1e71ae295fdb)

**Comment by cpietsch** [comment]
Still in `models`

**Reply by derpmagician** [reply]
Yeah Sorry, looks like i have a problem in my vscode that is hidding some of the folders on the Explore bar but i dont have that problem in its terminal

**Comment by ltdrdata** [comment]
If you didn't modified repository files.
just execute `git reset --hard && git pull`
